app/controllers/EventSummaryController.scala

```diff
@@ -52,8 +52,9 @@ class EventSummaryController @Inject()(
     (request.userAnswers.get(TaxYearPage), request.userAnswers.get(VersionInfoPage)) match {
       case (Some(taxYear), Some(versionInfo)) =>
         val startYear = s"${taxYear.startYear}-04-06"
-        connector.getEventReportSummary(request.pstr, startYear, versionInfo.version).map { seqOfEventTypes =>
-          seqOfEventTypes.map { es =>
+        connector.getEventReportSummary(request.pstr, startYear, versionInfo.version)
+          .map { seqOfEventTypes =>
+          seqOfEventTypes.collect { case es if !es.numberOfMembers.contains(0) =>
```

Comment:
please provide meaningful name instead of **es**

Reply:
Changed.